apps/browser/qml/pages/components/Overlay.qml

```diff
@@ -530,7 +558,8 @@ Shared.Background {
             Browser.FavoriteGrid {
                 id: favoriteGrid
 
-                height: historyList.height
+                y: historyButton.height
+                height: historyList.height - historyButton.height
```

Comment:
The underlying `IconGridViewBase` component adds extra margins to the left and right hand side, larger than the margin for other components in the toobar. This is accentuated with the addition of the History button.

Could we maybe have the `horizontalMargin: Theme.paddingLarge` set here so that they align?

Reply:
@llewelld they are not on the same level in design:
https://www.figma.com/file/boHTFFYJBsRQjZQjWispiJ/Browser?node-id=3743%3A1432

Reply:
But adding `horizontalMargin: Theme.paddingLarge` makes them look more like the design. Thanks.